cis/system/ksp-audit-postgresql-8-4.yaml

```diff
@@ -0,0 +1,20 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-audit-postgresql-8-4
+  namespace: default
+spec:
+  tags: ["PostgreSQL", "CIS", "rule-8.4"]
+  message: "Notification! PostgreSQL's Unix Socket Directories accessed"
+  selector:
+    matchLabels:
+      app: postgres 
+  file:
+    severity: 1
+    matchPath:
+    - path: /var/run/postgresql
```

Comment:
please check whether this is a folder or not and also add /tmp with recursive true, owneronly true @AbhinavCSY